### DISCUSSION TITLE
BelongsToMany does not shows on edit form

### DISCUSSION THREAD

**Discussion by 9350999** [laravel/nova-issues#6217] [Q&A] [answered]
- Laravel Version: 10.43.0
- Nova Version: v4.32.11
- PHP Version: 8.2.13
- Database Driver & Version: PostgreSQL v.14
- Operating System and Version: MacOS 13.5
- Browser type and version: Chrome: 121.0.6167.160 (Official Build) (arm64)
- Reproduction Repository: https://github.com/9350999/belongsToMany

### Description:
A field with the relationship `BelongsToMany` is only visible in the view form, but not in the edit form.

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Create two models with custom fields
2. To connect them together through a relationship `BelongsToMany`
3. Create a seeder and fill the database with the values of these models and create multiple connections between them
4. Create 2 resources for them including the field `BelongsToMany`
5. When opening the view form - related models will be visible in the table below the list element of the current model
6. When opening the edit form - the field `BelongsToMany` is not displayed

###Screenshots and video
<img width="1728" alt="Screenshot 2024-02-16 at 18 54 05" src="https://github.com/laravel/nova-issues/assets/82394997/107a7c19-9c57-4be8-bdb8-3b6775d0874c">

<img width="1727" alt="Screenshot 2024-02-16 at 18 56 17" src="https://github.com/laravel/nova-issues/assets/82394997/6799ccba-0070-4975-aeb3-bd9b74723ca4">

https://github.com/laravel/nova-issues/assets/82394997/89e549c3-13fb-4406-9d12-0f6572ea1668

**Comment by webard** [comment, marked as answer]
It is correct behaviour. Attaching relations is available after creating main entity.
